src/components/details.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Kindly do not disable linter because it enforces code quality. Instead, you should fix the issues raised.